news/2020-01-05-beatmap-spotlights-autumn-2019.md

```diff
@@ -0,0 +1,365 @@
+
+Even with these keymodes, these mappers combined incorporate these concepts on their own difficulties and arguably, the higher keymodes is where these concepts shine the most on, making this something we haven't quite seen in the ranked section before. With all that being said, the set is also keysounded, just to make your experience playing the set even more enjoyable than it is already.
+
+The set is incredibly dynamic, meticulously thought out, and even beautiful depending how you look at it and -MysticEyes makes all of this incredibly apparent through this particular mapset. If you're looking for a set which is a lot more toned down than your usual map and is very enjoyable to play, this set might be just for you.
+
+_written by [Unpredictable](https://osu.ppy.sh/users/7560872)_
+
+[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/mania/knife.jpg)](https://osu.ppy.sh/beatmapsets/809495)
+
+Even though this is [Cokiiplay](https://osu.ppy.sh/users/6760124)'s first bout in the ranked section, he is by no means a new face to the charting scene. He makes this abundantly clear in this chart for Knife that utilizes a fitting technical and meticulous approach. Within it are many elements that show off his charting experience such as deliberate placements of jumps, use of pitch relevance where applicable, and a clear level of repetition that fit extraordinarily well to both the song and the technical approach that was used.
+
+If that wasn't enough, there is also another layer to the meticulous charting scheme that Knife brings to the table, and that comes in how it is experienced while playing. Cokiiplay does a phenomenal job of combining his prowess with intuitive pattern motion that makes it feel like the player is actually making the drum and breakcore sounds in the song. Because of this, the playing experience is made to be extremely satisfying, an element that is very easy to miss when charting. The clear display of deliberate pattern choices with a playing experience that is arguably even better carves Knife a very deserving of his place in the Spotlights!
+
+_written by [Toaph Daddy](https://osu.ppy.sh/users/7616811)_
+
+---
+
+We've started to migrate over to our dedicated website properly in recent months, as our existing infrastructure was becoming increasingly tedious and outdated. In the near future, we'll have much more of what we do visible over at our brand new site, but for now, only the map selection is available. If you're interested, check it out [here](https://spotlights.team/spotlights-results).
+
+We hope you enjoy our selection for this season and look forward to what we will bring to you in the future. Until then, happy circle clicking, drum bashing, fruit catching, and key smashing! Do not forget to have a rest occasionally!
+
+—the Spotlights Team
```

Comment:
shouldn't this be signed Kurokami as the author of the post like previous spotlights.

Reply:
I am still not a fan of that, this is a team effort and not mine alone

Reply:
this part is just telling who wrote the uncredited parts of the post, not who contributed to the whole project

Reply:
I do not see it that way and i rather not want to be credited for literal copy pasting with some extra

Reply:
when we went back and changed all of the other authors, I think it was because the beginnings and ends of these posts are left with a false author---the whole team didn't write it, you did

in all fairness it's not really wrong to say "by the spotlights team" concerning the whole post cuz most of the team did write something. just feels weird to me cuz u already give credit to everyone else and the author doesn't mean anything more than who wrote the beginning/end

Reply:
Just because I add something to the beginning and at the end that doesn't make me author in my opinion. Look at it this way, everyone writes a tiny piece which means that it is done by the whole team, not just one person. Just because we write their names under each part that does not change this. And I was against the change in the previous Spotlights articles as well because of the same reason.

Reply:
I'm the person who started authoring posts "by the Project Loved team", I have looked at it this way before and that's what I'm saying in my second paragraph of last message. it turned out to be a bad idea because it falsely credited the whole team for the whole post, and nobody could tell who wrote the actual announcement beyond of each map description unless they checked github (we got questions about this for Loved which was why I looked into changing it back to names in the first place)

none of this is a big deal at all, but there are clear minor benefits to using your name as the author so I don't see why you shouldn't

Reply:
I do not mind if they do not know exactly who writes the announcement. I do not want to take credit for like 1% of the job when it mostly only consists of copy-pasting.

Reply:
I know you don't mind about that, I didn't either, but apparently some readers do. idk why you think of this as "taking credit" when all spotlights guys are already credited for exactly what they wrote. in the time we've been authoring team posts like this I've seen no complaints or confusion

Reply:
I never saw anyone complaining about this on Spots articles, they rather point out the small number of maps. 👀 The beginning and the end is so insignificant that they might just not even read it. I barely write anything and soon, thanks to Venix, even less.

Reply:
well I can't make you put ur name down if you don't want to, I think this is a strange decision considering how every other 'team' post is written